samples/unit-tests/series-scatter/general/demo.js

```diff
@@ -37,18 +38,17 @@ QUnit.test('Scatter series general tests.', function (assert) {
         '#15667).'
     );
 
+    tooltip.update({ animation: null });
```

Comment:
1. This should be (and is) done in [`test/karma-setup.js`](https://github.com/highcharts/highcharts/blob/master/test/karma-setup.js#L96). So you probably fixing here a side effect of another test.
2. It is better to figure out, which test is responsible for the side effect, and if the test is wrong, or if we need to improve the `test/karma-setup.js`. Hook some throw Error to the qunit afterEach function in karma-setup, that throws when the tooltip animation option has changed on the global scale.